### ISSUE TITLE
Refresh Keystroke?

### ISSUE THREAD

**Issue by WBAucoin** [arthurlockman/gatool-ui#373]
Consider adding a keystroke (F5?) to the gatool to enable the user to refresh the schedule, team information and/or current match and results via keystroke vs. either mouse or waiting.

**Comment by WBAucoin** [comment]
Where I would have liked this feature this past weekend was to quickly refresh the high scores. We had a high score in our finals and I wanted to see if it set a World Record. My memory is that we did (out doing the one set at Monterey Bay). In reality, we did not since that was recently "bested" by a different alliance competing on the same day. I expect we'll see that situation multiple times at the Houston CMP.

**Comment by WBAucoin** [comment]
There is a chance that this feature request may be OBE if you use the "auto refresh" feature (which currently checks every 15 seconds). Does the auto refresh also update the high scores? If not, could we do that?

**Comment by jameslockman** [comment]
World high scores refresh at the server level every hour or so. You won't be able to force that refresh, however it will appear within the hour. We fetch world high scores every time you get a schedule. Also, your results won't appear in gatool until they are posted to FIRST by the local field system, and after that, once they are made available by FIRST to API clients. This could be a long period of time, as we have seen all season.
On which screen(s) would you want this to be supported? 
We currently support the following gestures/keys:

- Swipe to go to next/previous match on Announce, Play By Play, and Emcee. 
- Right/left arrow to go to next/previous match on Announce, Play By Play and Emcee
- Shift-right arrow, period, and k to go to the next tab
- Shift-left arrow, comma, and j to go to the previous tab

**Comment by WBAucoin** [comment]
Thanks!

“World high scores refresh at the server level every hour or so.” = 😢

I guess we can pretend we set a new world record until someone informs us
of the recent newer high score.

**Comment by jameslockman** [comment]
I've added support for the s key and F5 to fetch schedules (with scores) and rankings for your event. We'll push this after other events complete today.
The reason we need to wait an hour for world high scores is that we read results from every event and comb through the results to find those three high scoring matches that we track. We used to do this as data flowed through our Schedule/scores API, but we missed results if there was no GA using gatool for the event, or if the user was offline during the event. While the delay is unfortunate, it mimics the existing delay at TBA. Depending on when your results post, the delay may be shorter.
